test/e2e/performance_test.go

```diff
@@ -190,25 +188,23 @@ func setupTestPods(data *TestData, b *testing.B) (nginxPodIP, abPodIP string) {
 		b.Fatalf("Error when waiting for IP assignment of nginx test Pod: %v", err)
 	}
 
-	b.Logf("Creating an apache-bench test Pod")
-	sleepDuration := "3600" // seconds
-	abPod := createPerformanceTestPodSpec(benchABPodName, abContainerName, abImage)
-	abPod.Spec.Containers[0].Command = []string{"sleep", sleepDuration}
-	_, err = data.clientset.CoreV1().Pods(testNamespace).Create(abPod)
+	b.Logf("Creating an perftool test Pod")
+	perfPod := createPerfTestPodDefinition(perftoolPodName, perftoolContainerName, perfToolImage)
+	_, err = data.clientset.CoreV1().Pods(testNamespace).Create(perfPod)
 	if err != nil {
-		b.Fatalf("Error when creating apache-bench test Pod: %v", err)
+		b.Fatalf("Error when creating perftool test Pod: %v", err)
 	}
-	b.Logf("Waiting IP assignment of the apache-bench test Pod")
-	abPodIP, err = data.podWaitForIP(defaultTimeout, benchABPodName)
+	b.Logf("Waiting IP assignment of the perftool test Pod")
+	perfPodIP, err = data.podWaitForIP(defaultTimeout, perftoolPodName)
 	if err != nil {
-		b.Fatalf("Error when waiting for IP assignment of apache-bench test Pod: %v", err)
+		b.Fatalf("Error when waiting for IP assignment of perftool test Pod: %v", err)
 	}
-	return nginxPodIP, abPodIP
+	return nginxPodIP, perfPodIP
 }
 
-// httpRequest runs a benchmark to measure intra-Node Pod-to-Pod HTTP request performance. It creates one Apache-Bench
+// httpRequest runs a benchmark to measure intra-Node Pod-to-Pod HTTP request perf. It creates one Apache-Bench
```

Comment:
you probably did a global s/performance/perf search and replace for the entire file, but in this comment I think keeping "performance" makes more sense :)

Reply:
Make sense, done.